sympy/solvers/tests/test_inequalities.py

```diff
@@ -323,15 +337,16 @@ def test_issue_10198():
         -1 + 1/abs(1/x - 1) < 0) == Or(
         And(-oo < x, x < 0), And(S(0) < x, x < S(1)/2)
         )
-    raises(NotImplementedError, lambda: reduce_inequalities(abs(1/sqrt(x)) - 1, x))
+    assert reduce_inequalities(abs(1/sqrt(x)) - 1, x) == Or(
+        And(-oo < x, x < 0), Eq(x, 1))
     assert reduce_abs_inequality(-3 + 1/abs(1 - 1/x), '<', x) == \
         Or(And(-oo < x, x < 0),
         And(S(0) < x, x < S(3)/4), And(S(3)/2 < x, x < oo))
     raises(ValueError,lambda: reduce_abs_inequality(-3 + 1/abs(
         1 - 1/sqrt(x)), '<', x))
 
 def test_issue_10047():
-    assert solve(sin(x) < 2) == And(-oo < x, x < oo)
+    assert solve(sin(x) < 2) == And(S(0) <= x, x < 2*pi)
```

Comment:
Why not define `solveset_univariate_inequality` for `solveset`, rather than modifying `solve_univariate_inequality` ?. To get complete solution we should not edit old one, right? For these changes it is fine (since it is returning soln within periodic interval), but for complete solution it will use `ImageSet`, isn't it ?